src/config.ts

```diff
@@ -1,5 +1,3 @@
-export const API_BASE_URL = 'https://content.jwplatform.com';
```

Comment:
If these are the same, let's drop the CONFIG from the naming and just call it `APP_API_HOST` for now (the only reason I made it config specific was that I wasn't sure if cdn.jwplayer.com and content.jwplatform.com were the same)